README.md

```diff
@@ -1,108 +1,204 @@
-    kind: ProviderConfig
-    metadata:
-      name: tenant-name
-    spec:
-      credentials:
-        secretRef:
-          namespace: crossplane-system
-          name: tenant-name-clouds-yaml
-          key: clouds.json
-        source: Secret
-    ---
-    apiVersion: v1
-    kind: Secret
-    metadata:
-      name: tenant-name-clouds-yaml
-      namespace: crossplane-system
-    stringData:
-      clouds.json: |
-        {
-          "auth_url": "https://api.gx-scs.sovereignit.cloud:5000",
-          "application_credential_id": "...",
-          "application_credential_secret": "...",
-          "tenant_name": "tenant-name"
-        }
-    ```
-1. Setup RBAC for tenants (wearing CSP hat)
-    ```yaml
-    apiVersion: rbac.authorization.k8s.io/v1
-    kind: ClusterRole
-    metadata:
-      name: tenant
-    rules:
-    - apiGroups:
-      - api.scs.community
-      resources:
-      - '*'
-      verbs:
-      - '*'
-    ---
-    apiVersion: v1
-    kind: ServiceAccount
-    metadata:
-      name: tenant
-      namespace: tenant-name
-    ---
-    apiVersion: rbac.authorization.k8s.io/v1
-    kind: RoleBinding
-    metadata:
-      name: scs-bind
-      namespace: tenant-name
-    roleRef:
-      apiGroup: rbac.authorization.k8s.io
-      kind: ClusterRole
-      name: tenant
-    subjects:
-    - kind: ServiceAccount
-      name: tenant
-      namespace: tenant-name
-    ```
-1. Create resource (wearing tenant hat, `kubectl --as system:serviceaccount:tenant-name:tenant -n tenant-name`)
-    ```yaml
-    apiVersion: api.scs.community/v1alpha1
-    kind: KeypairV2
-    metadata:
-      name: admin
-      namespace: tenant-name
-    spec:
-      name: admin-keypair
-      publicKey: |-
-        ssh-rsa ...
-    ---
-    apiVersion: api.scs.community/v1alpha1
-    kind: InstanceV2
-    metadata:
-      name: testing-vm
-      namespace: tenant-name
-    spec:
-      name: testing-vm
-      keyPair: admin-keyPair
-      imageName: 'Debian 12'
-      flavorName: 'SCS-1V:1:20'
-    ```
-1. Observe creation of resources
-
-Right now, it would be expected to hand out the `ServiceAccount` token to the actual tenant; When AuthN is done via OIDC (or other means), the `ServiceAccount` `tenant-name/tenant` may be dropped and `RoleBinding` `tenant-name/scs-bind` may point to an actual user/group.
+# SCS Central API
+
+## Premise
+
+By embracing existing open source solutions and bundling them, SCS is intended to
+provide a viable alternative to widely adopted proprietary cloud offerings, including
+Infrastructure-as-a-Service offerings, Kubernetes-as-a-Service offerings and other
+X-as-a-Service offerings.
+
+The choice to embrace existing technology has huge advantages over starting from
+scratch.
+By not reinventing wheels, a lot of effort can be saved. Existing communities can
+be strengthened. The adoption of existing open standards is supported, reducing
+market fragmentation and increasing interoperability.
```

Comment:
while this is imho a good intro for SCS in general I'm not 100% sure it's needed as an intro for the central api? I have no strong opinion on removing it though.

Reply:
I guess the "premise" section is not strictly required, but IMHO helps a lot clarifying scope and the current situation.

In particular, in relation to the central API, the first paragraph hints at...
- the scope of services within SCS, hence the expected scope of a central API
- the need to provide an UX on-par with hyperscalers, where central API's IMHO play an important role.

The second paragraph stresses the benefits of the chosen approach to build a cloud stack. Yes, technically, it's "fluff" (?), but I wanted to start off on a positive note here, as the "challenge" section highlights the problems in much more detail.

Reply:
i welcome that paragraph, to build on existing approaches has undoubtedly many advantages and it is a good starting point to assure that we are on the same page regarding this.